optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/chained/TestdataChainedEntity.java

```diff
@@ -39,7 +39,15 @@
         return entityDescriptor.getGenuineVariableDescriptor("chainedObject");
     }
 
+    public static GenuineVariableDescriptor<TestdataChainedSolution> buildVariableDescriptorForUnchainedObject() {
+        SolutionDescriptor<TestdataChainedSolution> solutionDescriptor = TestdataChainedSolution.buildSolutionDescriptor();
+        EntityDescriptor<TestdataChainedSolution> entityDescriptor = solutionDescriptor
+                .findEntityDescriptorOrFail(TestdataChainedEntity.class);
+        return entityDescriptor.getGenuineVariableDescriptor("unchainedObject");
+    }
+
     private TestdataChainedObject chainedObject;
+    private TestdataObject unchainedObject;
```

Comment:
(very soft) I 'd have used TestdataValue as a type for unchainedObject, calling it unchainedValue instead, for consistency with other testdata. But it doesn't matter much.